applications/MetisApplication/tests/test_MetisApplication.py

```diff
@@ -40,11 +41,8 @@ def AssembleTestSuites():
     # Create a test suit that contains all the tests from every testCase
     # in the list:
     allSuite = suites['all']
-    allSuite.addTests(
-        KratosUnittest.TestLoader().loadTestsFromTestCases([
-            KratosMetisGeneralTests
-        ])
-    )
+    allSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([KratosMetisGeneralTests]))
```

Comment:
```suggestion
    smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([KratosMetisGeneralTests]))
```

those tests should be fast, as mentioned please reduce the size of the mdpa

Reply:
I managed to reduce the mdpa size to 88Kb, it cannot be much smaller because what I am trying to check basically is that for each model part of the given submodelparts list, there is at least one entity on each processor (so the sub model part was effectively distributed between all processors) There can be a problem if submodelpart is very small and number of processors very high...